### ISSUE TITLE
Set coins throws error

### ISSUE THREAD

**Issue by GeorgeV220** [FyreHub/Fyreactyl#16]
When I'm trying to set coins by user email I got this error
```
ReferenceError: user_id is not defined
    at /handlers/api/email/set_coins.js:37:18
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by tovade** [comment]
Hi!

Will fix that soon!

Thanks for the bug report i haven't noticed myself

**Comment by tovade** [comment]
Fixed this in release 1.1.0, please use the latest version
